locales/es-ES/messages.po

```diff
@@ -25,18 +25,42 @@ msgstr ""
 msgid "%d characters allowed"
 msgstr ""
 
+msgid "%d Item(s) of %d imported from the given IMS QTI Package."
+msgstr ""
+
 msgid "%d/%d"
 msgstr ""
 
 msgid "%s metadata values found in source by extractor(s)."
 msgstr ""
 
+msgid "%s of %s characters maximum."
+msgstr "%s de %s caracteres máximo."
+
+msgid "%s of %s characters recommended."
+msgstr "%s de %s caracteres recomendados."
+
+msgid "%s of %s words maximum."
+msgstr "%s de %s palabras máximo."
+
 msgid "-- Any kind of file --"
 msgstr "--Cualquier tipo de archivo--"
 
 msgid "1 item imported from the given QTI/APIP XML Item Document."
 msgstr ""
 
+msgid "1/3 of height"
+msgstr ""
+
+msgid "1/4 of height"
+msgstr ""
+
+msgid "2/3 of height"
+msgstr ""
+
+msgid "3/4 of height"
+msgstr ""
```

Comment:
Are these used to set the height for an element or something like that, right? Like "Set the element height to 1/3 of height (of some other element/of the page's height)". If that's the case, I think these translations would work (if not, it may be more correct to use "1/3 de la altura").

```suggestion
msgid "1/3 of height"
msgstr "1/3 del alto"

msgid "1/4 of height"
msgstr "1/4 del alto"

msgid "2/3 of height"
msgstr "2/3 del alto"

msgid "3/4 of height"
msgstr "3/4 del alto"
```